docs/topics/settings.rst

```diff
@@ -889,8 +889,8 @@ DOWNLOAD_MAXSIZE
 
 Default: ``1073741824`` (1 GiB)
 
-The maximum response body size (in bytes) allowed. Bigger responses are
-aborted and ignored.
+The maximum response body size (in bytes) allowed. Bigger responses return
+:exc:`IgnoreRequest`.
```

Comment:
I think the previous wording was OK.

The fact that `IgnoreRequest` is raised feels to me like an internal detail, only relevant for downloader middleware writers, not for any user.

Reply:
How does this sound?

"The maximum response body size (in bytes) allowed. Responses greater in size are aborted and ignored."

Reply:
Sounds OK, but I don’t see it as an improvement over the current wording, it’s a bit lengthier and I don’t think it adds clarification to reword “Bigger” as “greater in size”. I still think it may be best to leave the original wording.